service/src/main/java/org/jboss/sbomer/service/feature/sbom/errata/dto/Errata.java

```diff
@@ -28,130 +30,105 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.JsonNode;
 
-import lombok.AllArgsConstructor;
-import lombok.Builder;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
-import lombok.NoArgsConstructor;
-import lombok.experimental.SuperBuilder;
 import lombok.extern.slf4j.Slf4j;
 
 @Data
 @Slf4j
-@Builder(setterPrefix = "with")
-@NoArgsConstructor
-@AllArgsConstructor
 public class Errata {
 
     @Data
-    @Builder(setterPrefix = "with")
-    @NoArgsConstructor
-    @AllArgsConstructor
     @JsonInclude(JsonInclude.Include.NON_NULL)
     public static class WrappedErrata {
-        Rhba rhba;
-        Rhea rhea;
-        Rhsa rhsa;
+        private Rhba rhba;
+        private Rhea rhea;
+        private Rhsa rhsa;
```

Comment:
Nitpicking -- shouldn't these be all upper case?

Reply:
right, fixing